### PR TITLE
Update code style

### PR DESCRIPTION
**Pull request by matt-oakes**
- Use `new` over `alloc init`
- Clarified comments
- Clarified literal
- Changed brase style

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,23 +103,21 @@ for (int i = 0; i < 10; i++) {
 ## Control Structures
 
  * Always surround control structure bodies with curly braces.
- * All curly braces should begin and end on a new line.
+ * Starting curly braces should begin on the same line as athe control structure and closing braces should be on their own line.
```
Comment (jemmaSlater) [inline README.md]: typo. "same line as _athe_ control"

Comment (manuelmarcos) [inline README.md]: Yahuuuu!!! Exciting times!!! (Sorry for the stupid comment)

**Comment by manuelmarcos** [comment]
Looks good to me. @jemmaSlater  feel free to merge in if you are happy too

File: README.md [hunk 2]
```diff
@@ -105,23 +103,21 @@ for (int i = 0; i < 10; i++) {
...
 }
 
-if (something == nil)
-{
+if (something == nil) {
     // do stuff
 }
-else
-{
+else {
```
Comment (jemmaSlater) [inline README.md]: I don't think this conforms to the guidelines we just agreed? Should be on the same line as the closing brace in the line above..

**Pull request merged**